assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -157,8 +202,14 @@ function FontPickerContainer({ value, onSelect, onClose }) {
 
   return (
     <PickerContainer ref={ref}>
+      <SearchInput
+        type="text"
+        value={searchKeyword}
+        onChange={handleSearchInputChanged}
+      />
       {matchingFonts.length ? (
         <List
+          onKeyDown={handleKeyPress}
```

Comment:
Hm, maybe we want to keep keyboard focus on the input field, while allowing arrow up/down to navigate the results?